addons/crm/static/src/views/crm_form/crm_pls_tooltip_button.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="crm.PlsTooltipButton">
-        <a class="o_crm_pls_tooltip_button border-0 mx-1 p-0 mb-1 mb-md-0 btn btn-light"
+        <a t-attf-class="o_crm_pls_tooltip_button border-0 mx-1 p-0 mb-1 mb-md-0 btn btn-light {{!this.props.record.resId ? 'invisible': ''}}"
```

Comment:
Hello @assh-odoo ,

Thanks for the fix. While this would work, could we keep the button to indicate the feature is available (as it was before) ? That way we do not add a component that is not shown in the view.

Atm, we allow saving before opening the tooltip in the JS, so could we try to modify `onClickPlsTooltipButton` as follows, instead of modifying the template, just after calling save on the record:
```js
if (status(this) === "destroyed" || !this.props.record.resId) {
    return;
}
```

Could you give it a go and see if everything works fine? Thanks! 

Cheers!

Reply:
Hello @Odoonan 

I have applied changes, and everything works fine.

Thanks!